editor/src/components/inspector/inspector-common.ts

```diff
@@ -725,26 +726,34 @@ export function setToFixedSizeCommands(
   })
 }
 
+const HuggingWidthHeightValues = ['max-content', 'min-content', 'fit-content', 'auto']
+
 export function isHugFromStyleAttribute(
   props: JSXAttributes,
   property: 'width' | 'height',
+  includeAllHugs: 'include-all-hugs' | 'only-max-content',
 ): boolean {
   const simpleAttribute = defaultEither(
     null,
     getSimpleAttributeAtPath(right(props), PP.create('style', property)),
   )
 
-  return simpleAttribute === MaxContent
+  if (includeAllHugs === 'only-max-content') {
+    return simpleAttribute === MaxContent
+  }
+
+  return simpleAttribute == null || HuggingWidthHeightValues.includes(simpleAttribute)
```

Comment:
`simpleAttribute == null` isn't right here. e.g. if we have:
```tsx
      <div
        style={{
          backgroundColor: 'red',
        }}
      >
        lalala
      </div>
```
The `div` will hug vertically, but will fill the parent horizontally.

Reply:
Also on line 755 below